Classes/Neos/Neos/Ui/Fusion/Helper/NodeInfoHelper.php

```diff
@@ -74,7 +74,11 @@ public function renderNode(NodeInterface $node, ControllerContext $controllerCon
             $nodeInfo['uri'] = $this->uri($node, $controllerContext);
         }
 
-        foreach ($node->getChildNodes() as $childNode) {
+        $documentChildNodes = $node->getChildNodes($this->baseNodeType);
+        $contentChildNodes = $node->getChildNodes('!Neos.Neos:Document');
```

Comment:
Why not `$node->getChildNodes('Neos.Neos:Content');` or (if this needs to fetch all other nodes) `$node->getChildNodes('!' . $this->baseNodeType);`?